docs/installation/considerations.mdx

```diff
@@ -0,0 +1,62 @@
+Die maximale Ladeleistung bzw. der maximale Strom lässt sich in den Wallboxen über Schalter konfigurieren. Der Installateur muss dies entsprechend der genehmigten Maximalleistung (Energieversorger), der Installation (Leitungsschutzschalter, FI-Schutz, Leitungsquerschnitt, Kabellänge, etc.) und dem verbauten CSS-Ladekabel korrekt einstellen. Fehler können hier schnell zu Ausfällen oder Kabelbränden führen. Im Schadensfall fragt die Versicherung nach dem Installationsprotokoll bzw. der Rechnung des vom Versorger zugelassenen Installateurs. 
+
+### Basis: Ohne Kommunikationsschnittstelle zur Steuerung des Ladevorgangs
+
+  Diese Wallboxen sind aufgrund der fehlenden Schnittstelle zur Steuerung des Ladevorgangs für das Überschussladen ungeeignet. Hier hilft nur, eine neue smarte oder professionelle Wallbox zu kaufen. In sehr, sehr seltenen Fällen können die Wallboxen mit einer Schnittstelle nachgerüstet werden (z.B. WARP2 Charger Basic auf WARP2 Charger Smart).
+
+### Smart: Mit Kommunikationsschnittstelle zur Steuerung des Ladevorgangs
+
+  Diese Wallboxen sind geeignet, sofern die [Schnittstelle von evcc unterstützt](/docs/devices/chargers) wird. Ein Indiz für eine Schnittstelle ist, wenn im Internet die Wallbox als förderfähig beworben wird. Als physische Schnittstelle kommt z.B. Ethernet, WLAN, LTE oder Modbus in Frage. Oftmals sind auch mehrere Schnittstellen gleichzeitig vorhanden. Zur Steuerung des Ladevorgangs bedarf es eines geeigneten Protokolls zwischen der Wallbox und evcc. Ein offener Standard, der sich mittlerweile weltweit etabliert hat, ist [OCPP (Open Charge Point Protocol)](https://de.wikipedia.org/wiki/OCPP). Sie sollten ggf. auf Unterstützung von OCPP Version 1.6 oder besser achten.
+
+  Neben der Kommunikation zwischen Wallbox und evcc ist die Kommunikation der Wallbox mit dem Fahrzeug relevant. Im einfachsten Fall kann die Wallbox nur den Ladestrom zwischen 6A und 32A signalisieren. Ja nachdem, ob die Wallbox mit einer oder mit drei Phasen an das Stromnetz angeschlossen wurde, lädt das Fahrzeug dann mindestens mit 1,38kW oder 4,14kW. Bei 11kW Wallboxen ist das Maximum bei einphasigem Anschluss dann bei 3,68kW erreicht. Um geringe Solarstromüberschüsse genauso wie größere Überschüsse laden zu können, empfiehlt sich eine automatische 1p/3p Umschaltung. Wallboxen, welche diese Funktion unterstützen sind in der [Aufstellung](/docs/devices/chargers) entsprechend vermerkt. Sollte die Wallbox eine automatische Umschaltung nicht unterstützen, könnte im Winterhalbjahr am vorgeschalteten Leitungsschutzschalter die Phase 2 und 3 ggf. händisch ausgeschaltet werden (sofern die Phasen überhaupt mechanisch einzeln zu schalten gehen und der FI-Schutzschalter mit nur einer Phase funktioniert).
```

Comment:
Die Verwendung von einzeln schaltbaren LS für dreiphasige Endgeräteleitungen ist keine (in Deutschland) zugelassene Kombination und auch ansonsten unbedingt zu vermeiden. LS-Automaten sind auch nicht für Schaltbetätigungen ausgelegt.

Hier ist die Verwendung eines Lasttrennschalters (z. B. Hager HAB304) für L2+L3 das richtige Mittel.

Reply:
OK, ich hatte diese Vermutung. Habe den Text auf Lasttrennschalter geändert.